tests/python_client/testcases/test_compaction.py

```diff
@@ -1238,7 +1237,7 @@ def do_index():
         replicas = collection_w.get_replicas()[0]
         replica_num = len(replicas.groups)
         seg_info = self.utility_wrap.get_query_segment_info(collection_w.name)[0]
-        assert len(seg_info) == 1*replica_num
+        assert len(seg_info) != 1*replica_num
```

Comment:
This case please modify the expected behavior in the comments, and describe clearly why it is changed.